src/build/dotnet/steps/net-framework.steps.yaml

```diff
@@ -0,0 +1,59 @@
+# Steps to build .NET Framework Projects
+parameters:
```

Comment:
do we need to continue supporting .net framework?

Reply:
Probably not, but I don't see harm in keeping this here in case we do.

Reply:
Have removed.